README.md

```diff
@@ -30,6 +30,7 @@ This is a collection of JS libraries to interact with the Hugging Face API, with
 
 - [@huggingface/inference](packages/inference/README.md): Use the Inference API to make calls to 100,000+ Machine Learning models, or your own [inference endpoints](https://hf.co/docs/inference-endpoints/)!
 - [@huggingface/hub](packages/hub/README.md): Interact with huggingface.co to create or delete repos and commit / download files
+- [@huggingface/agents](packages/agents/README.md): Interact with models through a natural language interface
```

Comment:
can be in another PR but let's also add a Usage example below!

Reply:
done!